src/sentry/static/sentry/app/actionCreators/teams.tsx

```diff
@@ -1,16 +1,38 @@
+import {Client} from 'app/api';
 import TeamActions from 'app/actions/teamActions';
 import {tct} from 'app/locale';
 import {addSuccessMessage, addErrorMessage} from 'app/actionCreators/indicator';
 import {uniqueId} from 'app/utils/guid';
+import {Team} from 'app/types';
+import {callIfFunction} from 'app/utils/callIfFunction';
 
-const doCallback = (params = {}, name, ...args) => {
-  if (typeof params[name] === 'function') {
-    params[name](...args);
-  }
+type CallbackOptions = {
+  success?: Function;
+  error?: Function;
 };
 
+const doCallback = (
+  params: CallbackOptions = {},
+  name: keyof CallbackOptions,
+  ...args
```

Comment:
any doesn't help much...but ...

```suggestion
  ...args: any[]
```